Cannot find module "config" upon install

When I try and install `zotra-cli` with `npm install -g .` in the cloned directory, it initially gave this error:

```
% zotra
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module 'config'
Require stack:
- /Users/yaqub/Library/Mobile Documents/com~apple~CloudDocs/Development/zotra-cli/bin/index.js
- ...
```

Note that the require stack contained *my personal development directory* that I cloned this repo to, not where my node modules are actually stored systemwide (`/opt/homebrew/lib/node_modules`.) In fact, if I go there, I do see a `zotra` directory, but it only contains the `src` folder, and thus only `attachments.js`. I am assuming this is not intended behaviour, so I added the following to package.json:

```
  "files": [
    "bin",
    "config",
    "module",
    "src"
  ],
```

After that, all of the folders were safely in the `zotra/` directory in node-modules, but when I ran `zotra` from the command line, it still got the above error (complete with a stack trace *still* referencing my development folder). Despite this, the actual `zotra` command itself seems to be referring now to the `bin/index.js` in my systemwide `node_modules`.

```
% which zotra
/opt/homebrew/bin/zotra
% ls -l /opt/homebrew/bin/zotra
lrwxr-xr-x  1 yaqub  admin  38 22 Aug 14:38 /opt/homebrew/bin/zotra -> ../lib/node_modules/zotra/bin/index.js
```

I am not sure what's going on here; either this is me screwing up (quite likely) or there is code in index.js that uses some kind of environment variable that isn't activated when npm is installed through homebrew, maybe?

I'm getting this error as well on a fresh install:

```shell
npm start

> zotra@1.0.0 start
> node bin/index.js server

node:internal/modules/cjs/loader:1404
  throw err;
  ^

Error: Cannot find module 'config'
Require stack:
- /home/cashweaver/proj/dotfiles/local/share/zotra-server/bin/index.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1401:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1057:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1062:22)
    at Function._load (node:internal/modules/cjs/loader:1211:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1487:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (/home/cashweaver/proj/dotfiles/local/share/zotra-server/bin/index.js:7:16)
    at Module._compile (node:internal/modules/cjs/loader:1730:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/cashweaver/proj/dotfiles/local/share/zotra-server/bin/index.js'
  ]
}

Node.js v22.15.0
```

--- a/package.json
+++ b/package.json
@@ -3,17 +3,18 @@
   "version": "1.0.0",
   "description": "A small wrapper around Zotero translation server to support fetching attachments and cli use",
   "main": "./bin/index.js",
-  "bin":{
-      "zotra": "./bin/index.js"
+  "bin": {
+    "zotra": "./bin/index.js"
   },
   "scripts": {
-      "start": "node bin/index.js server",
-      "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node bin/index.js server",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Mohammad Pedramfar",
   "license": "AGPLv3",
   "dependencies": {
-      "yargs": "^17.6.2",
-      "translation-server": "./module/translation-server"
+    "config": "^4.0.0",
+    "translation-server": "./module/translation-server",
+    "yargs": "^17.6.2"
   }
 }

